homeassistant/components/flux/switch.py

```diff
@@ -316,8 +316,10 @@ async def async_flux_update(self, utcnow=None):
                 self.hass, self._lights, x_val, y_val, brightness, self._transition
             )
             _LOGGER.debug(
-                "Lights updated to x:%s y:%s brightness:%s, %s%% "
-                "of %s cycle complete at %s",
+                (
+                    "Lights updated to x:%s y:%s brightness:%s, %s%% "
+                    "of %s cycle complete at %s"
+                ),
```

Comment:
Just my personal opinion: I'm not sure this kind of style change is actually an improvement. Guess if black wants to do it then we have to as well though 🤷🏻

Reply:
@cdce8p That was my initial reaction at first too...

However, there is logic to the madness that honestly does improve readability when having viewed a bunch of cases. I must be honest: I might actually start to appreciate it.

When a string is split over multiple lines, and the method also has other arguments, it becomes hard to see the difference between the multiple lines vs multiple arguments. In those cases, additional parentheses will be used to wrap the string to make it clear.

In all other cases (e.g., single line string with multiple additional arguments AND just single line string), these additional parentheses are actually omitted.

../Frenck

Reply:
> When a string is split over multiple lines, and the method also has other arguments, it becomes hard to see the difference between the multiple lines vs. multiple arguments (especially if a method has 2 multi-line strings in its call 😓 ). In those cases, additional parentheses will be used to wrap the string to make it clear.

I agree, those can be tricky sometimes.
Maybe I start to like this style as well, in time.

Thanks for the explanation!